metloom/pointdata/snotel.py

```diff
@@ -226,6 +239,32 @@ def _get_tzinfo(self):
             tz_hours = float(tz_hours)
         return timezone(timedelta(hours=tz_hours))
 
+    def _add_fixed_params(self, variables):
+        """
+        Get additional necessary fixed arguments for sensors that need heightDepth
+        params (soil moisture and soil temp)
+        """
+
+        extra_params = {}
+        for variable in variables:
+            if variable.name == self.ALLOWED_VARIABLES.TEMPGROUND2.name:
+                extra_params.update(
+                    {variable.name: {'height_depth': {"value": -2, "unitCd": "in"}}}
+                )
+            if variable.name == self.ALLOWED_VARIABLES.TEMPGROUND4.name:
+                extra_params.update(
+                    {variable.name: {'height_depth': {"value": -4, "unitCd": "in"}}}
+                )
+            if variable.name == self.ALLOWED_VARIABLES.TEMPGROUND8.name:
+                extra_params.update(
+                    {variable.name: {'height_depth': {"value": -8, "unitCd": "in"}}}
+                )
+            if variable.name == self.ALLOWED_VARIABLES.TEMPGROUND20.name:
```

Comment:
You should be able to drop `.name` from these comparisons (i.e. `if variable == self.ALLOWED_VARIABLES.TEMPGROUND20`)

If you want to simplify further I think the function could look like this
```

        extra_params_map = {
            self.ALLOWED_VARIABLES.TEMPGROUND2: {
                'height_depth': {"value": -2, "unitCd": "in"}
            },
            self.ALLOWED_VARIABLES.TEMPGROUND4: {
                'height_depth': {"value": -4, "unitCd": "in"}
            },
            self.ALLOWED_VARIABLES.TEMPGROUND8: {
                'height_depth': {"value": -8, "unitCd": "in"}
            },
            self.ALLOWED_VARIABLES.TEMPGROUND20: {
                'height_depth': {"value": -20, "unitCd": "in"}
            },

        }

        extra_params = {}
        for variable in variables:
            result = extra_params_map.get(variable)
            if result:
                extra_params[variable.name] = result

        return extra_params or None
```

Reply:
done!